content/docs/learning-guide/tutorials/ai/kythera-city-level-overview.md

```diff
@@ -0,0 +1,219 @@
+---
+linkTitle: Kythera City Level Overview
+title: Kythera City Level Overview
+description: Overview of the AI demo level "Kythera City"
+weight: 100
+---
+
+![A ship navigates Kythera City in 3D](/images/learning-guide/tutorials/ai/kythera-city-ship.png)
+
+The `KytheraCity` level demonstrates simple examples of setup for several Kythera AI features, in the Open3D (developers preview) build of the Kythera AI gem.
+
+The Kythera AI gem features demonstrated in this level are:
+
+*   [2D Navmesh generation](/docs/user-guide/gems/reference/kythera-ai/navmesh-generation) and customizability
+*   [3D Octree generation](/docs/user-guide/gems/reference/kythera-ai/octree-generation) and customizability
+*   [Kythera Inspector](/docs/user-guide/gems/reference/kythera-ai/introduction-to-the-inspector) [behavior trees](/docs/user-guide/gems/reference/kythera-ai/behavior-tree-editor)
+*   [Kythera Inspector Tool](/docs/user-guide/gems/reference/kythera-ai/introduction-to-the-inspector) debug draw
```

Comment:
```suggestion
* The [**Inspector**](/docs/user-guide/gems/reference/kythera-ai/introduction-to-the-inspector) tool, which includes:
    * [Behavior tree editing](/docs/user-guide/gems/reference/kythera-ai/behavior-tree-editor)
    * Debug drawing
```
To avoid stacking two links to the same 'Inspector' topic, consider combining the two items into one and nesting the two sub-features, like so.